AttitudeManager/ControlAlgorithms/Src/AM_FixedControl.cpp

```diff
@@ -9,7 +12,58 @@ std::vector<ActuatorOutput> FixedControl::runControlsAlgorithm(
     // interpret instructions
     // calculate pids
     // mix pids
-    // return output
+   // Get current attitude from sensorfusion 
+   
+    SFOutput_t current_attitude; 
+    
+    // Compute target values
+    float target_heading = instructions.heading;
+    float target_throttle = instructions.speed; 
+    float target_pitch = instructions.x_dir * MAX_PITCH_ANGLE;
+    float target_bank = instructions.y_dir * MAX_BANK_ANGLE;
```

Comment:
I just realized that  this probably wont allow us to calculate coordinated turns. Thoughts @antholuo @kozihkode @Gfcello? I'm not sure how AM might go about calculating this with just PIDs, unless I'm missing something

Reply:
I am also realizing that we might need to make explicit what x/y/z mean relative to the aircraft. It's not clear when just reading this code

Reply:
What happens when the input we receive is directly behind the aircraft?

Reply:
ok so, the issue here is the math that we're using to convert x/y/z vector. I'll talk to jack about this but GPS mode for a plane is very different for GPS mode for a quad....

We need to do a bit more calculations on the input vectors that basically sets a point x degrees away when we're in level mode, and when in gps mode we need to figure out hohw to fly to the point.

I believe what the code is right now looks like it's just giving passby manual control.

Reply:
"passby manual control" with a few PID's on each channel..... with no coordinated turning yet.

I'll look at this more in depth once AM is merged in